scripts/update_birdnet_snippets.sh

```diff
@@ -148,6 +148,9 @@ fi
 if ! grep FREQSHIFT_PITCH /etc/birdnet/birdnet.conf &>/dev/null;then
   sudo -u$USER echo "FREQSHIFT_PITCH=-1500" >> /etc/birdnet/birdnet.conf
 fi
+if ! grep FREQSHIFT_PITCH /etc/birdnet/birdnet.conf &>/dev/null;then
```

Comment:
Is this meant to be checking if `ACTIVATE_FREQSHIFT_IN_LIVESTREAM` exists and not `FREQSHIFT_PITCH` in birdnet.conf?

Reply:
FREQSHIFT_PITCH value is only for the "sox" tool, if this last one is chosen by the user for freq shifting of mp3 or wav files.
ACTIVATE_FREQSHIFT_IN_LIVESTREAM is an option for adding freq shifting option to "ffmpeg" in livestream.sh. So you are right, before activating this option, we must take care to FREQSHIFT_LO and FREQSHIFT_HI, and look if they exist. But normally this is already the case